services/docker-compose.yml

```diff
@@ -229,7 +229,7 @@ services:
         - traefik.http.routers.${SWARM_STACK_NAME}_webserver.rule=hostregexp(`{host:.+}`)
           && (Path(`/`, `/v0`,`/socket.io/`,`/static-frontend-data.json`,
           `/study/{study_uuid:\b[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}\b}`,
-          `/view`) ||  PathPrefix(`/v0/`))
+          `/view`, `/#/view`) ||  PathPrefix(`/v0/`))
```

Comment:
@sanderegg I am afraid is not the last ... viewers (``studies_dispatcher`` plugin) as a redirection syntax that uses fragments to communicate with the front end.
For instance ``https://osparc.io/#/error?message=Sorry%2C%20I%20could%20not%20find%20this%20&status_code=404``

Reply:
Actually we use fragments for much more at the root as well
```
https://osparc.io#reset-password?code=131234
https://osparc.io#reset-password?registered
https://osparc.io/#/registration/?invitation={code}
```

Reply:
ok, so for the /view and /error I will set on top /view/#/ and /error/#/. for the last 3 ones I'm puzzled.